### DISCUSSION TITLE
What is a "zone"?

### DISCUSSION THREAD

**Discussion by 0xUniko** [ProjectOpenSea/seaport#560] [Q&A] [answered]
There is a zone data field in OrderFulfilled event. I do not quite understand this concept. Is there any code example to show that how to calculate or obtain this zone value?

**Comment by arr00** [comment, marked as answer]
Read about zones in the [documentation](https://github.com/ProjectOpenSea/seaport/blob/main/docs/SeaportDocumentation.md#order). A zone is an ethereum address to either an EOA or a contract which can have special privileges on the order as described in the docs.
